packages/next-codemod/transforms/next-image-experimental.ts

```diff
@@ -196,7 +201,7 @@ function nextConfigTransformer(j: JSCodeshift, root: Collection) {
         const normalizeSrc = `const normalizeSrc = (src) => src[0] === '/' ? src.slice(1) : src`
         if (loaderType === 'imgix') {
           writeFileSync(
-            filename,
+            join(appDir, filename),
```

Comment:
I think you should pass also `parsed.root` (check the docs here https://nodejs.org/api/path.html#pathparsepath for explaination)

Reply:
@grzpab Can you give an example where it would fail as-is?

It looks like `parsed.dir` is all thats needed according to the docs 

<img width="308" alt="image" src="https://user-images.githubusercontent.com/229881/219693048-d42be0c5-6d37-4275-9b1f-15ecbbe707c3.png">

Reply:
In fact, the docs show here that `dir` always includes `root` so it shouldn't be needed


<img width="309" alt="image" src="https://user-images.githubusercontent.com/229881/219693260-0c64c443-7524-4482-b18f-a36645079889.png">